examples/pytorch_from_centralized_to_federated/client.py

```diff
@@ -29,10 +29,12 @@ def __init__(
         model: cifar.Net,
         trainloader: torch.utils.data.DataLoader,
         testloader: torch.utils.data.DataLoader,
+        num_examples: dict(),
```

Comment:
```suggestion
        num_examples: Dict,
```

(same as above, `Dict` needs to be imported from `typing`)